nerfstudio/models/splatfacto.py

```diff
@@ -186,6 +186,29 @@ def __init__(
     ):
         self.seed_points = seed_points
         super().__init__(*args, **kwargs)
+        self.selected_gaussian_indices = []
+        self.original_colors = None
+
+    def update_colors(self):
```

Comment:
I think this function is self-contained enough that it can be defined inside `on_select`, that way its scope is clearer (it only gets used inside that callback).